### PR TITLE
feat(trace) add continuous profile link

### PR DESCRIPTION
**Pull request by JonasBa**
Add a temporary link to the UI so that we can link to continuous profiles. This is not the final place for the button, but it's placed in a convenient top level so that we can quickly move back and forth

### CODE REVIEW

File: static/app/views/performance/newTraceDetails/traceDrawer/traceProfilingLink.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ function toDate(value: unknown): Date | null {
  */
 export function makeTraceContinuousProfilingLink(
   value: TraceTreeNode<TraceTree.NodeValue>,
-  transaction: EventTransaction,
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/traceProfilingLink.ts:29]: Modified this so we dont tie it to txn even anymore (it is now also available on the spans)

File: static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx [hunk 2]
```diff
@@ -536,15 +560,32 @@ function SectionCard({
...
+  ${CardPanel} {
+    container-type: inline-size;
+  }
+
+  ${Subject} {
+    @container (width < 350px) {
+      max-width: 200px;
+    }
+  }
+`;
```
Comment (Zylphrex) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx:587]: Should this be part of this PR?

**Review by Zylphrex — APPROVED** [review]

File: static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx (hunk 2 above)
Comment (JonasBa) [reply static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx:587]: Oh no, I need to remove this, I ammended a commit and messed it up

Comment (JonasBa) [reply static/app/views/performance/newTraceDetails/traceDrawer/details/styles.tsx]: No, I removed it. 4 day weekends hit hard

**Pull request merged**